### DISCUSSION TITLE
fresh install Updates?

### DISCUSSION THREAD

**Discussion by mexsudo** [SpiralLinux/SpiralLinux-project#310] [Q&A]
I downloaded the ISO this morning... 
first thing I did was open synaptic, refreshed  and checked for updated packages
I find this very odd. unless I use a net install there is always something needed since the ISO was created.

Question: is the ISO continually updated?

**Comment by geckolinux** [comment]
Hi there, nope, the ISO is usually not updated frequently. But since this latest release was generated only 2 days ago from the latest Debian sources there are no new updates since then.

**Reply by mexsudo** [reply]
ok, thanks. I thought I might have overlooked a setup step. 

however I did just get my first update

**Comment by mexsudo** [comment]
I do appreciate the rapid and clear response.
thanks, Peter
